tests/test_autonomy/test_cli/test_push_all.py

```diff
@@ -53,7 +54,9 @@ def test_run(
     ) -> None:
         """Test run."""
 
-        packages = set([file.parent for file in self.t.glob("**/*yaml")])
+        # packages/<author>/<component>/<name>/<config>.yaml
+        path = f"{PACKAGES}/*/*/*/*.yaml"
+        packages = {file.parent for file in self.t.glob(path)}
```

Comment:
needed because of the nested skill.yaml in the `tests/data/dummy_abci`. Since we strictly follow the structure in the comment this is arguably a better way to test this as well.